CONTRIBUTING.md

```diff
@@ -0,0 +1,51 @@
+# Contributing to Eclipse zenoh
+
+Thanks for your interest in this project.
+
+## Project description
+
+Eclipse zenoh provides is a stack  designed to 
+  1. minimize network overhead, 
+  2. support extremely constrained devices, 
+  3. supports devices with low duty-cycle by allowing the negotiation of data exchange modes and schedules, 
+  4. provide a rich set of abstraction for distributing, querying and storing data along the entire system, and 
+  5. provide extremely low latency and high throughput.
+
+* https://projects.eclipse.org/projects/iot.zenoh
+
+## Developer resources
+
+Information regarding source code management, builds, coding standards, and
+more.
+
+* https://projects.eclipse.org/projects/iot.zenoh/developer
+
+The project maintains the following source code repositories
+
+* https://github.com/eclipse-zenoh
+
+## Eclipse Contributor Agreement
+
+Before your contribution can be accepted by the project team contributors must
+electronically sign the Eclipse Contributor Agreement (ECA).
+
+* http://www.eclipse.org/legal/ECA.php
+
+Commits that are provided by non-committers must have a Signed-off-by field in
+the footer indicating that the author is aware of the terms by which the
+contribution has been provided to the project. The non-committer must
+additionally have an Eclipse Foundation account and must have a signed Eclipse
+Contributor Agreement (ECA) on file.
+
+For more information, please see the Eclipse Committer Handbook:
+https://www.eclipse.org/projects/handbook/#resources-commit
+
+## Contact
+
+Contact the project developers via the project's "dev" list.
+
+* https://accounts.eclipse.org/mailing-list/zenoh-dev
+
+Or via the Gitter channel.
+
+* https://gitter.im/atolab/zenoh
```

Comment:
Where did you copy this file from? The link to gitter is very old and everything has now moved to Discord...

Reply:
Oops..... that's the DDS bridge :)
Most of the typical stuff is taken from there

Reply:
It needs to be update here and also in the zenoh-plugin-dds repository then...

Reply:
I'll do that

Reply:
mqtt plugin too....